style.css

```diff
@@ -0,0 +1,692 @@
+body {
+  font-family: Lato, sans-serif;
+  margin: 0;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  background: #f8f8f8;
+}
+
+.wrapper {
+  padding: 0 15% 10%;
+  display: flex;
+  flex-direction: column;
+}
+
+.headline {
+  background-image: url("Img/bg-desktop.png");
+  background-repeat: no-repeat;
+  padding-top: 1%;
+  background-position-x: center;
+  height: 90vh;
+}
+
+.into > h3 {
+  font-weight: 800;
+  color: #747679;
+  line-height: 50px;
+  font-size: 2rem;
+}
+
+.into > h6 {
+  text-align: center;
+}
+
+.intro-email a {
+  font-weight: 600;
+  color: #424247;
+  display: flex;
+  justify-content: space-around;
+}
+
+.into > h2 {
+  font-weight: 900;
+  color: #ec5242;
+  line-height: 50px;
+  font-size: 3rem;
+  width: 55%;
+  animation: 1s ease-out 0s 1 slideInFromLeft;
+}
+
+.into > h1 {
+  font-weight: 400;
+  color: #ec5242;
+  line-height: 40px;
+  font-size: 2rem;
+}
+
+.into > p {
+  line-height: 24px;
+  background-color: #fafafa;
+  border: 4px solid #fffcfc;
+  padding: 5%;
+  width: 65%;
+  animation: 1s ease-out 0s 1 slideInFromRight;
+}
+
```

Comment:
- [ ] As requeted by the previous reviewer, kindly make sure the paragraph container has a `white` background color on the `About` page in order to match the provided Behance design.

  - ### YOUR DESIGN
  ![image](https://github.com/obibaadoma/captone_DaaNkwa/assets/87197412/9bea3678-eead-4ce4-9969-c157b2e5224c)

  - ### BEHANCE DESIGN

   ![image](https://github.com/obibaadoma/captone_DaaNkwa/assets/87197412/437f4bc9-1d38-40ef-a58f-02df82870e86)